src/js/base/post-load-hooks.js

```diff
@@ -0,0 +1,136 @@
+          booleans_only : runtime.getField(ds, "booleans-only"),
+          empty_only : runtime.getField(ds, "empty-only")
+        };
+
+        runtime["makeCStr"] = runtime.getField(ds, "c-str").app;
+        runtime["makeCNum"] = runtime.getField(ds, "c-num").app;
+        runtime["makeCBool"] = runtime.getField(ds, "c-bool").app;
+        runtime["makeCCustom"] = runtime.getField(ds, "c-custom").app;
+        runtime["makeCEmpty"] = function() { return runtime.getField(ds, "c-empty"); };
+
+        runtime["isCStr"] = function(v) { return runtime.unwrap(runtime.getField(ds, "is-c-str").app(v)); };
+        runtime["isCNum"] = function(v) { return runtime.unwrap(runtime.getField(ds, "is-c-num").app(v)); };
+        runtime["isCBool"] = function(v) { return runtime.unwrap(runtime.getField(ds, "is-c-bool").app(v)); };
+        runtime["isCCustom"] = function(v) { return runtime.unwrap(runtime.getField(ds, "is-c-custom").app(v)); };
+        runtime["isCEmpty"] = function(v) { return runtime.unwrap(runtime.getField(ds, "is-c-empty").app(v)); };
+
+        runtime["unwrapCellContent"] = function(v) {
+          if (runtime.isCStr(v)) {
+            return {type: "str", value: runtime.getField(v, "s")};
+          } else if (runtime.isCNum(v)) {
+            return {type: "num", value: runtime.getField(v, "n")};
+          } else if (runtime.isCBool(v)) {
+            return {type: "bool", value: runtime.getField(v, "b")};
+          } else if (runtime.isCCustom(v)) {
+            return {type: "custom", value: runtime.getField(v, "datum")};
+          } else if (runtime.isCEmpty(v)) {
+            return {type: "empty"};
+          } else {
+            runtime.ffi.throwMessageException("Internal error: Cannot unwrap non-cell content");
+          }
+        };
+
+        runtime["makeLoadedTable"] = function(headers, contents) {
+          // Headers can either be [name, sanitizer] arrays or
+          // {name: name, sanitizer: sanitizer} objects
+          headers = headers.map(function(h) {
+            if (h.sanitizer) {
+              return runtime.makeTuple([h.name, h.sanitizer]);
+            } else {
+              return runtime.makeTuple(h);
+            }
+          });
+          return runtime.makeTuple([headers, contents]);
+        };
+        runtime["checkCellContent"] = runtime.makeCheckType(
+          runtime.getField(ds, "is-CellContent").app, "CellContent");
+      },
+      "builtin://reactors": function(reactor) {
+        var r = runtime.getField(runtime.getField(reactor, "provide-plus-types"), "values");
+        runtime.setParam("makeReactor", runtime.getField(r, "make-reactor").app);
+      },
+      "builtin://checker": function(checker) {
+        checker = runtime.getField(runtime.getField(checker, "provide-plus-types"), "values");
+        // NOTE(joe): This is the place to add checkAll
+        var currentChecker = runtime.getField(checker, "make-check-context").app(runtime.makeString(main), true);
```

Comment:
The last parameter is a variable called `checkAll` in `load-lib`, and is unconditionally `true` here

Reply:
Fixed, thanks.